src/ansys/aedt/core/filtersolutions_core/distributed_radial.py

```diff
@@ -0,0 +1,287 @@
+        self._dll.setDistributedFixedAngle.restype = int
+        self._dll.getDistributedFixedAngle.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedFixedAngle.restype = int
+
+        self._dll.setDistributedDeltaStubs.argtype = c_bool
+        self._dll.setDistributedDeltaStubs.restype = c_int
+        self._dll.getDistributedDeltaStubs.argtype = POINTER(c_bool)
+        self._dll.getDistributedDeltaStubs.restype = c_int
+
+        self._dll.setDistributedSplitWideAngle.argtype = c_char_p
+        self._dll.setDistributedSplitWideAngle.restype = c_int
+        self._dll.getDistributedSplitWideAngle.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedSplitWideAngle.restype = c_int
+
+        self._dll.setDistributedOffsetFromFeedline.argtype = c_char_p
+        self._dll.setDistributedOffsetFromFeedline.restype = c_int
+        self._dll.getDistributedOffsetFromFeedline.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedOffsetFromFeedline.restype = c_int
+
+        self._dll.setDistributedAlternateRadialDeltaOrientation.argtype = c_bool
+        self._dll.setDistributedAlternateRadialDeltaOrientation.restype = c_int
+        self._dll.getDistributedAlternateRadialDeltaOrientation.argtype = POINTER(c_bool)
+        self._dll.getDistributedAlternateRadialDeltaOrientation.restype = c_int
+
+        self._dll.setDistributedAdjustWidthMax.argtype = c_bool
+        self._dll.setDistributedAdjustWidthMax.restype = c_int
+        self._dll.getDistributedAdjustWidthMax.argtype = POINTER(c_bool)
+        self._dll.getDistributedAdjustWidthMax.restype = c_int
+
+        self._dll.setDistributedMaxRadialDeltaAngle.argtype = c_char_p
+        self._dll.setDistributedMaxRadialDeltaAngle.restype = c_int
+        self._dll.getDistributedMaxRadialDeltaAngle.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedMaxRadialDeltaAngle.restype = c_int
+
+        self._dll.setDistributedAdjustLengthMax.argtype = c_bool
+        self._dll.setDistributedAdjustLengthMax.restype = c_int
+        self._dll.getDistributedAdjustLengthMax.argtype = POINTER(c_bool)
+        self._dll.getDistributedAdjustLengthMax.restype = c_int
+
+        self._dll.setDistributedMinRadialDeltaAngle.argtype = c_char_p
+        self._dll.setDistributedMinRadialDeltaAngle.restype = c_int
+        self._dll.getDistributedMinRadialDeltaAngle.argtypes = [c_char_p, c_int]
+        self._dll.getDistributedMinRadialDeltaAngle.restype = c_int
+
+        self._dll.setDistributedApplyLimitsRadialDelta.argtype = c_bool
+        self._dll.setDistributedApplyLimitsRadialDelta.restype = c_int
+        self._dll.getDistributedApplyLimitsRadialDelta.argtype = POINTER(c_bool)
+        self._dll.getDistributedApplyLimitsRadialDelta.restype = c_int
+
+    @property
+    def radial_stubs(self) -> bool:
+        """Flag indicating if the distributed radial stubs is enabled.
+        This property utilizes radial stubs for open lines.
```

Comment:
Will this wording be ok?
```suggestion
        If true, radial stubs are used for open lines.
```